src/finch/julia.py

```diff
@@ -1,6 +1,9 @@
 import juliapkg
 
-juliapkg.add("Finch", "9177782c-1635-4eb9-9bfb-d9dfa25e6bce", version="0.6.20")
+deps = juliapkg.deps.load_cur_deps()
+if not deps.get("packages", {}).get("Finch", {}).get("version", None) == "0.6.20":
+    juliapkg.add("Finch", "9177782c-1635-4eb9-9bfb-d9dfa25e6bce", version="0.6.20")
```

Comment:
I'd define `_FINCH_VERSION` and `_FINCH_HASH` and use those -- so it doesn't need updating in two places.